### PR TITLE
PCC52 hobojoe1848

### PR DESCRIPTION
**Pull request by hobojoe1848**
ATTENTION: before clicking "Create Pull Request" please submit some meta data, thanks!

Difficulty level (1-10): [3]
Estimated time spent (hours): [1]
Completed (yes/no): [Yes]
I stretched my coding skills (if yes what did you learn?): [I learned to keep it simple and keep each function to one task only.]
Other feedback (what can we improve?): []

### CODE REVIEW

**Comment by pybites** [comment]
No Flask? ;)

**Comment by pybites** [comment]
Love that you did "1 hour + KISS + PR", instead of "no time + no PR"

File: 52/hobojoe1848/pomodoro.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+import time
+
+DURATION = 10
+
+def pomodoro(duration):
+    print(f'Starting Pomodoro timer for {duration} minutes.')
+    print(f'Start time: {current_time()}')
```
Comment (pybites) [inline 52/hobojoe1848/pomodoro.py]: f-strings 👍

File: 52/hobojoe1848/pomodoro.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    time.sleep(duration*60)
+    print(f'End time: {current_time()}')
+
+def current_time():
+    return((datetime.now().strftime("%Y-%m-%d %H:%M")))
```
Comment (pybites) [inline 52/hobojoe1848/pomodoro.py]: don't need the double (( ))

**Comment by hobojoe1848** [comment]
> No Flask? ;)

HAHA! No Flask! KISS for this one :P

- Thanks! Loving f strings these days!

- Good catch! Completely missed the extra bracket set on the return!

**Pull request merged**